### DISCUSSION TITLE
Getting Outline to work the way I like to work - to fork or not to fork!

### DISCUSSION THREAD

**Discussion by r-colvin** [outline/outline#4074] [General]
@Limezy continuing our discussion here: https://github.com/outline/outline/discussions/4072

the question was "what 10% of my needs is not met"

I keep a single note for every day; and I capture the full days work in that note. This works well for me as it enables me to capture anything and any time. If I am in a meeting, and something triggers inspiration, I can quickly capture it without having to find the right note, tab or whatever. It gets from my brain to my note quickly. Having a note-per-day allows me to keep a chronological order which is important for me. For capturing this flow works very well for me. However, it is terrible for information retrieval - it is painful to know where I was in a task, what i planned to do next... etc. If i go on vacation for a few weeks, I have to go digging for where I was. This ends up in information, tasks etc getting lost, or massive amounts of duplication. If i work on the same task/project for a few days in a row, i can generally work ok but after that its a pain.

I have been working on this for years - i have read many books, looked into many work flows and tried more tools than I care to remember; evernote, logseq, obsidian, roam, one note, joplin, things3, todoist, bear,  custom markdown files... and im sure there are many more that I just don't remember. I have tried adjusting my work flow to the way the tool works and favours but at the end i _always_ come back to my capture system. In terms of priority - i find that capturing fast is my highest prio and I use search tools, pinned notes, and other tools to make finding the info I need easier. For big things I have a whiteboard at my desk where i note things down as well.

For the record then, I currently use Evernote. I personally really don't like Evernote as a tool! I use it because I like the Editor. I can paste screenshots (which I do a LOT), i can embed media, the keyboard shortcuts - like date insert are helpful. There is a mobile app (even if cloud sync sucks). So for me, the Editor is the best editor that I have found, and I work well with it. Thus Evernote is the least bad option I have found and why I stick with it.

Outline does most of the things that Evernote does - I took it for a spin as a fill in for my daily note taking - the missing features in the editor are things that others have requested; resizing media like images, and youtube embeds, font size control - honestly I want the Evernote editor! Outline's editor is so close though that I can survive some features - a CSS hack brings the embedded media to 400px and that works, same with default font size. I have Evernote zoomed out a few clicks to make better use of my 24" monitor - and i prefer a smaller font size to maximise screen real estate - though I like the 3 pane layout of Evernote.

over the years I have collected a fairly extensive collection of use-cases I want my note taking system to support, but i will be a bit more high level here. But suffice to say, I have a very good idea of what I want as I have long planned on building my own - using the Evernote API for example so I can use it's editor but my own front end! 

Where Outline (and Evernote and everything else ive tried) falls down. In my day-per-note I break things into sections using a bold-heading. then i collect everything as list-items under that. This might be work on a project, meeting notes or whatever. So a section starts with a bold heading (the project name) immediately under is the time and then a list. What I want is to have my day-notes in a single folder, and then I want virtual collections/notebooks so that I can view one project in its entirety, so when I come back, i can go to the project view, quick review and then back to work. 

this is a small selection of the tool of my dreams - ive spoken to friends who have the same problems and we have workshopped various use cases. Hence I was so excited to see that Outline's editor was open source, but disappointed that it's archived. To be honest, although I could probably build my own front end (i have considered doing this with evernote) i don't really want to... building a note taking system is not my primary interest - i want it to support the other things I do. Hence I am looking at self-hosting outline... which i might do, as it's better than Evernote - but what I really want is the ability to build virtual notebook/collections. Now if pulling content from a single note is a pain, the other solution ive considered is to just create a new note for each "section" so maybe a note called "project-202209071158" then the virtual notebook could look at the note title instead ... or maybe set a tag on the note for the time when I want a note to be in multiple collections (for example a meeting may discuss multiple projects, so I want the meeting notes in two collections)

hrmm... hope I have explained myself. As I say, I've been looking for a solution for nearly 10 years... and nothing has shown up. So i figure i'm on my own!

**Comment by tommoor** [comment]
You _could_ use the API for this, set up some sort of cron job to read your daily created note from the API, parse the headings and then write the content from each heading to different documents/collections. There are also webhooks so you could do this in near-realtime with a little more complexity.
